src/renderer/shared/ui/Tabs/Tabs.tsx

```diff
@@ -43,7 +43,7 @@ export const Tabs = ({
     </Tab.List>
     <Tab.Panels className={panelClassName}>
       {items.map(({ id, panel }) => (
-        <Tab.Panel key={id} unmount={unmount}>
+        <Tab.Panel tabIndex={-1} key={id} unmount={unmount}>
```

Comment:
To remove redundant focus of content area?

Reply:
It's not useful. Also I found that `Tabs.Panel` are really bad in combination with `ScrollArea`, so `VotesModal` use Tabs only as navigation switch